README.md

```diff
@@ -9,7 +9,7 @@ The default encoding for the models is JSON, although in at least one case --
 YAML encoding is also supported since a device profile is defined as a YAML document.
 
 ### Installation ###
-* Make sure you're using at least Go 1.11.1 and have modules enabled, i.e. have an initialized  go.mod file 
+* Make sure you're using at least Go 1.11.1 (EdgeX currently uses Go 1.15.2) and have modules enabled, i.e. have an initialized  go.mod file 
```

Comment:
we might just state 1.15, because we don't define the patch number in the go.mod and Dockerfile

Reply:
in edgex-go, we are using 1.15.x